json/build.sh

```diff
@@ -55,3 +55,20 @@ cd json-clj; lein uberjar; cd ..; cp json-clj/test.jar ./
 # python/python3/pypy
 pip install ujson
 pip3 install ujson
+
+# java
+mkdir json-java
+cp TestJava.java json-java/
+cd json-java
+wget https://repo1.maven.org/maven2/com/dslplatform/dsl-json/0.9.1/dsl-json-0.9.1.jar
+wget https://repo1.maven.org/maven2/com/dslplatform/dsl-clc/1.5.0/dsl-clc-1.5.0-jar-with-dependencies.jar
+wget https://repo1.maven.org/maven2/com/dslplatform/dsl-json-processor/0.1/dsl-json-processor-0.1.jar
+javac -Adsljson.namespace=bench -processor com.dslplatform.json.CompiledJsonProcessor -cp dsl-json-0.9.1.jar:dsl-json-processor-0.1.jar:dsl-clc-1.5.0-jar-with-dependencies.jar TestJava.java
+mkdir META-INF
+mkdir META-INF/services
+mv com.dslplatform.json.Configuration META-INF/services
+mkdir bench
+mv Test*.class bench
+mkdir bench/json
+mv External*.class bench/json
+cd ..
```

Comment:
thanks, but maybe some easier way to compile install it, some java package manager?